kubernetes/application/README.md

```diff
@@ -0,0 +1,62 @@
+# Blueprint Name
+
+## Introduction
+
+Use this blueprint to ...
+
+## Before you get started
+
+If you're new to XebiaLabs blueprints, check out:
+
+* [Get started with DevOps as Code](https://docs.xebialabs.com/xl-platform/concept/get-started-with-devops-as-code.html)
+* [Get started with blueprints](https://docs.xebialabs.com/xl-platform/concept/get-started-with-blueprints.html)
+* [Get started with XL JetPack](https://docs.xebialabs.com/xl-platform/concept/get-started-with-xl-jetpack.html)
+
+## Usage
+
+To use this blueprint, run `xl blueprint` and select:
+
+    [full name of blueprint in the CLI menu]
+
+## Tools and technologies
+
+This blueprint includes the following tools and technologies:
+
+* Target:
+    * [targets that the blueprint will deploy to]
+* Tools:
+    * [tools that the blueprint invokes]
+* Application or framework:
+    * [application that the blueprint deploys, if applicable]
+
+## Minimum required versions
+
+This blueprint version requires at least the below versions of the specified tools to work properly.
```

Comment:
"The below versions" doesn't sound like native US English to me. I would write "the versions below".

Reply:
that was part of the templates, so fixed em all (including template for future)